docker/elipselink/README.md

```diff
@@ -0,0 +1,51 @@
+# Polaris with EclipseLink, Postgres and Spark SQL
+
+1. If such an image is not already present, build the Polaris image with support for EclipseLink and
+   the Postgres JDBC driver:
+
+    ```shell
+    ./gradlew clean :polaris-quarkus-server:assemble :polaris-quarkus-admin:assemble \
+       -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 \
+       -Dquarkus.container-image.build=true
+    ```
+
+2. Start the docker compose group using either podman or docker:
+
+    ```shell
+    podman compose -f docker/eclipselink/docker-compose.yml up
+    docker compose -f docker/eclipselink/docker-compose.yml up
```

Comment:
nit: making separate blocks of text makes copy-paste into shell easier -- GH renders the copy widget, but it copies the whole block of text :)